bosh-create-env-deployment.yml

```diff
@@ -112,7 +114,9 @@ jobs:
       port: 25777
       cloud:
         plugin: aws
-        credentials_source: env_or_profile
+        credentials_source: static
+        access_key_id: (( grab meta.aws.access_key_id ))
```

Comment:
I know this is hypocritical, since you just asked me the same question about some other deployments, but wdyt about merging these from terraform?